yarn-project/builder/src/cli.ts

```diff
@@ -43,14 +26,16 @@ const main = async () => {
     .argument('[projectPath]', 'Path to the project directory', process.cwd())
     .option('--contract [paths...]', 'Paths to contracts to update dependencies', [])
     .option('--aztec-version <semver>', 'The version to update Aztec packages to. Defaults to latest', 'latest')
-    .addOption(pxeOption)
```

Comment:
This option was unused and it might have caused the issue. I'll test if it resolved it once the new docker image gets published. I didn't manage to build the docker image locally because I suck at devops.